ext/xml/tests/xml_parser_get_option_variation4.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+xml_parser_get_option() - Test that a parameter not set defaults
```

Comment:
The contents of the TESTS section are printed whenever the test is run; I suggest to keep that a single line. If more descriptive text is needed, it could be placed in a [DESCRIPTION section](http://qa.php.net/phpt_details.php#description_section), or just added as comment to the FILE section.

Reply:
Done, thanks for pointing this out.